projects/1.20/assets/xtra-arrows/xtraarrows/lang/zh_cn.json

```
@@ -0,0 +1,305 @@
+    "entity.xtraarrows.netherite_slime_arrow": "史莱姆下界合金箭",
+    "entity.xtraarrows.iron_slime_arrow": "史莱姆铁箭",
+    "entity.xtraarrows.golden_slime_arrow": "史莱姆金箭",
+    "entity.xtraarrows.flint_slime_arrow": "史莱姆箭",
+
+    "entity.xtraarrows.diamond_ender_arrow": "末影钻石箭",
+    "entity.xtraarrows.netherite_ender_arrow": "末影下界合金箭",
+    "entity.xtraarrows.golden_ender_arrow": "末影金箭",
+    "entity.xtraarrows.iron_ender_arrow": "末影铁箭",
+    "entity.xtraarrows.flint_ender_arrow": "末影箭",
+
+    "entity.xtraarrows.diamond_tracking_arrow": "追踪钻石箭",
+    "entity.xtraarrows.netherite_tracking_arrow": "追踪下界合金箭",
+    "entity.xtraarrows.golden_tracking_arrow": "追踪金箭",
+    "entity.xtraarrows.iron_tracking_arrow": "追踪铁箭",
+    "entity.xtraarrows.flint_tracking_arrow": "追踪箭",
+
+    "entity.xtraarrows.diamond_vexing_arrow": "恼鬼钻石箭",
+    "entity.xtraarrows.netherite_vexing_arrow": "恼鬼下界合金箭",
+    "entity.xtraarrows.golden_vexing_arrow": "恼鬼金箭",
+    "entity.xtraarrows.iron_vexing_arrow": "恼鬼铁箭",
+    "entity.xtraarrows.flint_vexing_arrow": "恼鬼箭",
+
+    "entity.xtraarrows.diamond_soul_torch_arrow": "灵魂火把钻石箭",
+    "entity.xtraarrows.flint_soul_torch_arrow": "灵魂火把箭",
+    "entity.xtraarrows.netherite_soul_torch_arrow": "灵魂火把下界合金箭",
+    "entity.xtraarrows.iron_soul_torch_arrow": "灵魂火把铁箭",
+    "entity.xtraarrows.golden_soul_torch_arrow": "灵魂火把金箭",
+
+    "entity.xtraarrows.diamond_atlantean_arrow": "海神钻石箭",
+    "entity.xtraarrows.netherite_atlantean_arrow": "海神下界合金箭",
+    "entity.xtraarrows.golden_atlantean_arrow": "海神金箭",
+    "entity.xtraarrows.iron_atlantean_arrow": "海神铁箭",
+    "entity.xtraarrows.flint_atlantean_arrow": "海神箭",
+
+    "entity.xtraarrows.diamond_freezing_arrow": "冰冻钻石箭",
+    "entity.xtraarrows.netherite_freezing_arrow": "冰冻下界合金箭",
+    "entity.xtraarrows.golden_freezing_arrow": "冰冻金箭",
+    "entity.xtraarrows.iron_freezing_arrow": "冰冻铁箭",
+    "entity.xtraarrows.flint_freezing_arrow": "冰冻箭",
+
+    "entity.xtraarrows.diamond_magnetic_arrow": "磁吸钻石箭",
+    "entity.xtraarrows.iron_magnetic_arrow": "磁吸铁箭",
+    "entity.xtraarrows.flint_magnetic_arrow": "磁吸箭",
+    "entity.xtraarrows.golden_magnetic_arrow": "磁吸金箭",
+    "entity.xtraarrows.netherite_magnetic_arrow": "磁吸下界合金箭",
+
+    "entity.xtraarrows.diamond_life_steal_arrow": "生命偷取钻石箭",
+    "entity.xtraarrows.iron_life_steal_arrow": "生命偷取铁箭",
+    "entity.xtraarrows.flint_life_steal_arrow": "生命偷取箭",
+    "entity.xtraarrows.golden_life_steal_arrow": "生命偷取金箭",
+    "entity.xtraarrows.netherite_life_steal_arrow": "生命偷取下界合金箭",
+
+    "entity.xtraarrows.diamond_gravity_arrow": "失重钻石箭",
+    "entity.xtraarrows.iron_gravity_arrow": "失重铁箭",
+    "entity.xtraarrows.flint_gravity_arrow": "失重箭",
+    "entity.xtraarrows.golden_gravity_arrow": "失重金箭",
+    "entity.xtraarrows.netherite_gravity_arrow": "失重下界合金箭",
+
+    "effect.xtraarrows.magnetized" : "磁化",
+
+    "itemGroup.xtraarrows": "额外箭矢",
+
+    "xtraarrows.text.netherite_damage": "\u00A7o\u00A79+6 伤害",

```

```suggestion
    "xtraarrows.text.netherite_damage": "\u00A7o\u00A79+6伤害",
```
若无格式要求，请注意半角字符和全角字符间不需空格，全文通查